opflexagent/config.py

```diff
@@ -57,6 +57,15 @@
     cfg.StrOpt('fabric_bridge', default='br-fabric',
                help=_("The name of the bridge which connects to the ACI "
                       "fabric")),
+    cfg.StrOpt('bridge_manager',
+               default=('opflexagent.utils.bridge_managers.'
+                        'ovs_manager.OvsManager'),
```

Comment:
Since we are making a pluggable manager, should we use entry points instead of paths?

Reply:
Good suggestion -- will fix.